doc/classes/FileAccess.xml

```diff
@@ -540,6 +592,42 @@
 				[b]Note:[/b] If an error occurs, the resulting value of the file position indicator is indeterminate.
 			</description>
 		</method>
+		<method name="store_u8">
+			<return type="bool" />
+			<param index="0" name="value" type="int" />
+			<description>
+				Stores an unsigned integer as 8 bits in the file.
+				[b]Note:[/b] The [param value] should lie in the interval [code][0, 255][/code]. Any other value will overflow and wrap around.
+				[b]Note:[/b] If an error occurs, the resulting value of the file position indicator is indeterminate.
+			</description>
+		</method>
+		<method name="store_u16">
+			<return type="bool" />
+			<param index="0" name="value" type="int" />
+			<description>
+				Stores an unsigned integer as 16 bits in the file.
+				[b]Note:[/b] The [param value] should lie in the interval [code][0, 2^16 - 1][/code]. Any other value will overflow and wrap around.
+				[b]Note:[/b] If an error occurs, the resulting value of the file position indicator is indeterminate.
+			</description>
+		</method>
+		<method name="store_u32">
+			<return type="bool" />
+			<param index="0" name="value" type="int" />
+			<description>
+				Stores an unsigned integer as 32 bits in the file.
+				[b]Note:[/b] The [param value] should lie in the interval [code][0, 2^32 - 1][/code]. Any other value will overflow and wrap around.
+				[b]Note:[/b] If an error occurs, the resulting value of the file position indicator is indeterminate.
+			</description>
+		</method>
+		<method name="store_u64">
+			<return type="bool" />
+			<param index="0" name="value" type="int" />
+			<description>
+				Stores an unsigned integer as 64 bits in the file.
+				[b]Note:[/b] The [param value] must lie in the interval [code][0, 2^63 - 1][/code] (i.e. be a valid [int] value).
```

Comment:
```suggestion
				[b]Note:[/b] The [param value] must lie in the interval [code][0, 2^64 - 1][/code]. Since [int] is signed, values ​​above [code]2^63 - 1[/code] cannot be represented correctly (will be negative).
```

Reply:
Not sure what's the best way to convey it, it might require a bit more detailed description to avoid confusion.